Source/Core/loadImageFromTypedArray.js

```diff
@@ -37,7 +37,8 @@ define([
             request: request
         });
         return resource.fetchImage({
-            flipY : flipY
+            flipY : flipY,
```

Comment:
Above - should the default be `false`?

`var flipY = defaultValue(options.flipY, true);`